Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\n# as it contains information specific to your local configuration.\n#\n# Location of the SDK. This is only used by Gradle.\n# For customization when using a Version Control System, please read the\n# header note.\n#Fri Jul 15 11:20:40 IST 2022\n#sdk.dir=/home/waheguru/Android/Sdk\nsdk.dir=/Users/johndoe/Library/Android/sdk\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision 58e0abf2d3a3f582cf71a3966f08c05c99515960)
+++ b/local.properties	(date 1662273607293)
@@ -6,5 +6,5 @@
 # header note.
 #Fri Jul 15 11:20:40 IST 2022
 #sdk.dir=/home/waheguru/Android/Sdk
-sdk.dir=/Users/johndoe/Library/Android/sdk
+sdk.dir=/home/waheguru/Android/Sdk
 
